app/main.py

```diff
@@ -1,13 +1,34 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return (
+        {
+            "errors": [],
```

Comment:
If the format_linter_error function does not find any errors, it will return an empty list as well. Therefore, you only need to adjust the part that may differ, specifically the 'status' logic.
Exp : 'status' : 'failed" if ... else ...